ui/packages/app/src/project_setting/project_info/UpdateProjectInfoModal.js

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect } from "react";
+import { addToast, useMlpApi } from "@caraml-dev/ui-lib";
+import {
+  EuiConfirmModal,
+  EuiOverlayMask,
+  EuiCode,
+  EuiBasicTable,
+  EuiFlexItem,
+  EuiFlexGroup
+} from "@elastic/eui";
+import { useNavigate } from "react-router-dom";
+
+const UpdateProjectInfoModal = ({
+  project,
+  closeModal,
+  fetchUpdates,
+  originalProject
+}) => {
+  const [submissionResponse, submitForm] = useMlpApi(
+    `/v1/projects/${project.id}`,
+    {
+      method: "PUT",
+      headers: { "Content-Type": "application/json" }
+    },
+    {},
+    false
+  );
+
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (submissionResponse.isLoaded && !submissionResponse.error) {
+      closeModal();
+      addToast({
+        id: "submit-success-create",
+        title: "Project Info Updated!",
+        color: "success",
+        iconType: "check"
+      });
+      fetchUpdates();
+      navigate(`/projects/${submissionResponse.data.id}/settings/project-info`);
+    }
+  }, [navigate, submissionResponse, project, fetchUpdates, closeModal]);
+
+  const handleUpdate = () => {
+    const updatedProjectInfo = {
+      ...project,
+      stream: project.stream,
+      team: project.team,
+      labels: project.labels
+    };
+
+    submitForm({ body: JSON.stringify(updatedProjectInfo) });
+  };
+
+  const columns = [
+    {
+      field: "key",
+      name: "Key"
+    },
+    {
+      field: "value",
+      name: "Value"
+    }
+  ];
+
+  const rows = project.labels.map(label => ({
+    ...label
+  }));
+
+  const originalRows = originalProject.labels.map(label => ({
+    ...label
+  }));
+
+  return (
+    <EuiOverlayMask>
+      <EuiConfirmModal
+        title="Update Project Info"
+        onCancel={() => closeModal()}
+        onConfirm={handleUpdate}
+        cancelButtonText="Cancel"
+        confirmButtonText="Update"
+        buttonColor="primary"
+        defaultFocusedButton="confirm">
+        <p>
+          You are about to update the project info for project{" "}
```

Comment:
Can we modify this to, "You are about to update the project info for **the** project..."

Reply:
Ok will do!